osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -20,20 +22,119 @@ public class Speed : Skill
         private const double pi_over_4 = Math.PI / 4;
         private const double pi_over_2 = Math.PI / 2;
 
+        /// <summary>
+        /// streamaimconst is derived from the last stream of Megalovania [Excors] 9* map (Unit:pixel/ms)
+        /// </summary>
+        private const double streamaimconst = 2.42;
+
+        /// <summary>
+        /// splitter value for jumpDistances2 list
+        /// </summary>
+        private int sdsplitcounter;
```

Comment:
These xmldocs are useless, they tell me nothing more than the variable name itself. Don't restate things in xmldoc.

Reply:
I think someone might confuse the 1,2 number iterations trust me, I'll leave it there.